src/map/utils/charutils.cpp

```diff
@@ -2915,7 +2915,7 @@ namespace charutils
                     }
                     else if (PetID == PETID_CAIT_SITH)
                     {
-                        if (PAbility->getID() > ABILITY_SOOTHING_RUBY && PAbility->getID() <= ABILITY_MOONLIT_CHARGE)
+                        if (PAbility->getID() > ABILITY_SOOTHING_RUBY && PAbility->getID() < ABILITY_MOONLIT_CHARGE)
```

Comment:
Why was this changed? Was it necessary?

Reply:
yes, when cait sith was out, moonlit charge was in the list of usable blood pacts